src/security.rst

```diff
@@ -0,0 +1,74 @@
+Security
+========
+
+TODO: Fill in other parts of this document, with information about
+cylc-flow (jobs, zmq, etc), UI Server, etc.
+
+Web
+---
+
+TODO: write some introduction about web security in Cylc 8,
+UI Server, OAuth, JupyterHub Authenticators, etc.
```

Comment:
@oliver-sanders and @kinow I guess these TODOs have mostly been addressed by #285. But is there anything else in this old PR that you think we should still add to the docs?

Reply:
There are a little more detail here I think, but the #285 contains links to JupyterHub docs. The detailed information here could be (or become) outdated if JupyterHub changes how they store cookies (or dropped cookies), and manage websockets.

Might be easier to leave the high-level security information in the new docs, and increase the detail if necessary.

Besides the details, #285 appears to cover everything here, including the TODO's that were planned for later :tada: 

Only thing I'd ask before closing, is whether we should have a page only about security. Searching "security" on GitHub, I found it in the architecture, the summary page, and others. But normally decision-makers, security engineers, some IT operations engineers, etc, may prefer to have a single place to refer about security.

Or maybe we already have it somewhere else that I am not aware @hjoliver ? WDYT?

Reply:
Yeah we should have a security page in the docs, even it mainly just links elsewhere.  I'll post an Issue for that, and close this one.